tests/main/op-remove-retry/task.yaml

```
@@ -5,6 +5,7 @@ systems: [-ubuntu-14.04*]
 
 restore: |
     kill %1 || true
+    pkill -f "sleep 9999" || true
```

This maybe needs a comment explaining why we need to kill it
```suggestion
    # sleep was started from a snap by the unmount-blocker unit, normally
    # it should be enough to kill the whole unit, however since that process
    # was started from a snap, it got moved to a separate snap-specific transient
    # scope, thus it is no longer part of the unit's cgroup
    pkill -f "sleep 9999" || true
```

As a followup, what if we added a test level cleanup code to find non empty cgroups that match a pattern `snap.*.<uuid>.scope`?